src/main/java/frc/robot/subsystems/boathook/Boathook.java

```diff
@@ -168,8 +191,39 @@ public void addInstruments() {
 
   @Override
   public void periodic() {
+    // if (rotationMotor.getReverseLimit().getValue() == ReverseLimitValue.ClosedToGround) {
+    //   rotationEncoder
+    //       .getConfigurator()
+    //       .apply(
+    //           rotationEncoderConfig.MagnetSensor.withMagnetOffset(
+    //               rotationEncoder.getPosition().getValue()));
+    //   rotationMotor
+    //       .getConfigurator()
+    //       .apply(this.resetMinToZero(rotationConfig, rotationMotor.getPosition().getValue()));
+    // }
+
+    // if (extenderMotor.getReverseLimit().getValue() == ReverseLimitValue.ClosedToGround) {
+    //   extensionEncoder
+    //       .getConfigurator()
+    //       .apply(
+    //           extensionEncoderConfig.MagnetSensor.withMagnetOffset(
+    //               extensionEncoder.getPosition().getValue()));
+    //   //   extenderMotor
+    //   //       .getConfigurator()
+    //   //       .apply(this.resetMinToZero(extenderConfig,
+    // extenderMotor.getPosition().getValue()));
+    // }
+
     // This method will be called once per scheduler run
     SmartDashboard.putNumber("Boathook Angle", getAngle());
     SmartDashboard.putNumber("Boathook Extension", getLength());
   }
+
+  private TalonFXConfiguration resetMinToZero(TalonFXConfiguration configuration, Angle angle) {
+    return configuration
+        .withFeedback(new FeedbackConfigs().withFeedbackRotorOffset(angle))
+        .withHardwareLimitSwitch(
+            new HardwareLimitSwitchConfigs().withReverseLimitAutosetPositionEnable(false));
+  }
+  ;
```

Comment:
If the commented code above goes, this method should go too. It's only used there.